### PR TITLE
Fix/centos build

### PR DESCRIPTION
**Pull request by a-zakir**

### CODE REVIEW

File: docker/centos7-build [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+
+RUN mkdir -p /workspace/vcpkg_cache
+ARG VCPKG_CACHE_DIR
+COPY $VCPKG_CACHE_DIR /workspace/
+
+
+ARG BRANCH=develop
+RUN cd /workspace && \
+    source /opt/rh/rh-git227/enable && \
+    git clone https://github.com/AntaresSimulatorTeam/antares-xpansion.git --branch $BRANCH && \
```
Comment (flomnes) [inline docker/centos7-build]:
```suggestion
    git clone https://github.com/AntaresSimulatorTeam/antares-xpansion.git --depth 1 --branch $BRANCH && \
```

Comment (flomnes) [reply docker/centos7-build]: Use shallow clone to reduce bandwidth & disk usage 🍃

File: docker/centos7-build [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+    wget $URL_ANTARES && \
+    tar -xvf antares-${ANTARES_VERSION}-CentOS-7.9.2009.tar.gz -C deps --strip-components=1 &&\
+    rm -rf antares-${ANTARES_VERSION}-CentOS-7.9.2009.tar.gz
+
+WORKDIR /workspace/antares-xpansion
+
+RUN pip3 install --upgrade pip && \
+    pip3 install wheel && \
+    pip3 install -r requirements-tests.txt && \
+    pip3 install -r requirements-ui.txt
```
Comment (flomnes) [inline docker/centos7-build:50]: pip [accepts multiple arguments](https://stackoverflow.com/a/36744306)

```suggestion
    pip3 install -r requirements-tests.txt -r requirements-ui.txt
```

**Review by flomnes — COMMENTED** [review]
LGTM, a few remarks

**Comment by flomnes** [comment]
Also there seems to be an issue with PyQT5 being incompatible with python3.9 on Windows & Ubuntu, so you have to take care of this too

**Comment by a-zakir** [comment]
> Also there seems to be an issue with PyQT5 being incompatible with python3.9 on Windows & Ubuntu, so you have to take care of this too

unrelated to this pr, in fact the centos build succeed

File: docker/centos7-build (hunk 2 above)
Comment (a-zakir) [reply docker/centos7-build:50]: good to know

**Review by flomnes — APPROVED** [review]

**Pull request merged**